src/Hamiltonians/ReducedDensityMatrix.jl

```diff
@@ -0,0 +1,150 @@
+"""
+    SingleParticleReducedDensityMatrix(i, j) <: AbstractHamiltonian
+
+Represent a {i,j} element of the single-particle reduced density matrix:
```

Comment:
Don't like how this reads. Maybe
```suggestion
Represent the ``(i, j)``-th element of the single-particle reduced density matrix:
```

Reply:
Thank you, @mtsch, for your review. But it seems that you checked the old version of this pull request code before I made all the necessary changes. Please check the latest version of the code.